src/foam/dao/AuthenticatedDAO.java

```diff
@@ -97,29 +100,34 @@ public FObject find_(X x, Object id) {
 
   @Override
   public Sink select_(X x, Sink sink, long skip, long limit, Comparator order, Predicate predicate) {
-    // x.get("auth") != null added so that system user can act during booting when the auth service is not loaded yet.
-    if ( authenticateRead_ && x.get("auth") != null ) {
-      super.select_(x, new AuthenticatedSink(x, createPermission("read"), sink), skip, limit, order, predicate);
+    if ( authenticateRead_ ) {
```

Comment:
So you no longer have the `x.get("auth") != null` check anywhere right? I 100% support that because I think it could lead to security problems, however the comment implies that we needed it during boot?  Is that no longer the case?

Reply:
I think so. I removed it and started the server and everything seemed to be working. I could sign in to the portal. Then I tried running the test suite and it passed. So based on that I'm assuming we don't need it anymore.